lldb/source/Plugins/ExpressionParser/Clang/ClangExpressionParser.cpp

```diff
@@ -561,7 +562,85 @@ static void SetupLangOpts(CompilerInstance &compiler,
   lang_opts.NoBuiltin = true;
 }
 
-static void SetupImportStdModuleLangOpts(CompilerInstance &compiler) {
+// NOTE: should be kept in sync with sdkSupportsBuiltinModules in
+// Toolchains/Darwin.cpp
+static bool
+sdkSupportsBuiltinModulesImpl(const llvm::Triple &triple,
```

Comment:
Can you move this into `XcodeSDK.h`?

Reply:
Like make it a `static` function on `XcodeSDK`? We're not creating any `XCodeSDK` instances here (since we're not actually parsing the `DW_AT_LLVM_sysroot` here)

Reply:
Yeah, even if it's a static method it'd be good to centralize that logic. If someone needs something similar in the future, it's the first place they'd look.

Reply:
Tbh I'd prefer keeping this as an implementation detail. We really don't want to duplicate this logic. And can hopefully remove it eventually. If it lives as an API in a header that has a slight chance of making it harder to remove.
But lmk if either of you feel strongly about it.